nifi-nar-bundles/nifi-smb-bundle/nifi-smb-nar/pom.xml

```diff
@@ -35,5 +35,16 @@
             <artifactId>nifi-smb-processors</artifactId>
             <version>1.17.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-smbj-client</artifactId>
+            <version>1.17.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
The processor nar should not depend on a controller service implementation. As far as I can see, it is not used and can just be deleted.

Reply:
@kulikg Please delete this dependency. Otherwise the controller service is duplicated on the UI.